src/components/layout/DefaultWindow.tsx

```diff
@@ -0,0 +1,66 @@
+import { FC, ReactNode } from 'react';
+
+import SvgSprite from 'components/shared/SvgSprite';
+import 'styles/components/default-window.scss';
+
+interface Data {
+  message: string;
+  documentation_url: string;
+}
+
+interface Error {
+  data: Data;
+  status: number;
+}
+
+interface Props {
+  loading: boolean;
+  error?: Error;
+}
+const DefaultWindow: FC<Props> = props => {
+  const { loading = false, error } = props;
+
+  const sizeIcon = 70;
+  const defaultText = 'Start with searching a GitHub user';
+
+  const getLoader = (): ReactNode | null => {
```

Comment:
create global type at global.d.ts 

```suggestion
  type Nullable<T> = T | null


  const getLoader = (): Nullable<ReactNode> => {
```